src/gestion_aula.py

```diff
@@ -60,8 +60,12 @@ def esta_en_clase(alumno_entrada, clase):
 
 
 def seleccionar_alumno_random(clase):
-    indice_random = random.randint(0, len(clase) - 1)
-    return clase[indice_random]
+    try:
+        indice_random = random.randint(0, len(clase) - 1)
+        return clase[indice_random]
+    except ValueError:
+        print("No hay alumnos en aún.")
+        nuevo_alumno()
```

Comment:
Estoy de acuerdo en comprobar que no este la clase vacía pero yo no crearía un alumno nuevo en esta situación

```suggestion
    except ValueError:
        print("La clase esta vacía.")
```